spring-cloud-zuul-ratelimit-core/src/main/java/com/marcosbarbero/cloud/autoconfigure/zuul/ratelimit/config/properties/RateLimitProperties.java

```diff
@@ -62,6 +62,7 @@
     private Map<String, List<Policy>> policyList = Maps.newHashMap();
     private boolean behindProxy;
     private boolean enabled;
+    private boolean showHeader = true;
```

Comment:
I think a name like `addResponseHeaders` would be cleaner on what this property does, WDYT?

Reply:
you're right